calculatrice/src/App.js

```diff
@@ -7,54 +7,30 @@ function App() {
 	const [total, setTotal] = useState("0");
 	const [operator, setOperator] = useState(null);
   const [number, setNumber] = useState("0");
-	const [result, setResult] = useState("0")
+	// const [result, setResult] = useState("0")
 
-	const addOperator = (operatorSign) => {
-			switch (operator) {		
-				case Operator.PLUS:
-					setTotal(total + parseFloat(number));
-					break;
-
-				case Operator.MINUS:
-					setTotal(total - parseFloat(number));
-					break;
-
-				case Operator.MULTIPLY:
-					setTotal(total * parseFloat(number));
-					break;
-
-				case Operator.DIVIDE:
-					setTotal(total / parseFloat(number));
-					break;
-
-				default:
-					setTotal(parseFloat(number));
-			}
-
-      setNumber("0");
-			setOperator(operatorSign);
-	}
-
-  const doCalculation = (input) => () => {
-    const num = parseFloat(number);
-		
-    if (input === "AC") {
-      setNumber("0");
-      setResult("0");
+  const doAC = () => {
+    setNumber("0");
+      // setResult("0");
       setTotal("0");
       setOperator(null);
       return;
-    }
+  }
+
+  const checkCommas = () => {
 
-    if (input === ".") { 
-      if (number.includes(".")) return; //Checks for multiple inputs of commas
+    if (number.includes(".")) return;
 
       setNumber(number + ".");
       return;
-    }
+  }
+
+  const addInput = (input) => () => {
+
+    const num = parseFloat(number);
 
     if (input === Operator.PLUS) {
-      addOperator(Operator.PLUS) //Makes sure we have a previous operation
+      addOperator(Operator.PLUS) 
       return;
     }
     if (input === Operator.MINUS) {
```

Comment:
comme je disais l'autre jour, cette forêt de if serait plus lisible en un switch case